src/ui/public/field_editor/field_editor.html

```diff
@@ -92,22 +92,32 @@ <h4 class="hintbox-heading">
         ng-model="editor.field.count"
         type="number"
         class="form-control"
-        data-test-subj=editorFieldCount>
+        data-test-subj=editorFieldCount
+      >
 
-      <span class="input-group-btn">
+      <span class="kuiButtonGroup kuiButtonGroup--united">
```

Comment:
This one has focus issues, and maybe looks a little weird with the curved corners on the lefT:
<img width="429" alt="screen shot 2017-04-21 at 10 15 29 am" src="https://cloud.githubusercontent.com/assets/16563603/25281548/bb681b72-267b-11e7-8961-44c6675b770c.png">

<img width="438" alt="screen shot 2017-04-21 at 10 15 49 am" src="https://cloud.githubusercontent.com/assets/16563603/25281563/c53202da-267b-11e7-8b33-11b921688dd6.png">